### DISCUSSION TITLE
How to enable Split Diff view?

### DISCUSSION THREAD

**Discussion by victor-falcon** [jesseduffield/lazygit#3941] [Q&A] [answered]
I saw this in the latest releases, how can I view diffs this way in lazyvim?

![image](https://github.com/user-attachments/assets/39082952-2586-4a4a-a46e-10705df73ad6)

**Comment by mark2185** [comment, marked as answer]
It's a [custom pager](https://github.com/jesseduffield/lazygit/blob/master/docs/Custom_Pagers.md), but I'm not sure which one. Looks like `delta`?
